lib/pages/chat/empty_chat_view.dart

```diff
@@ -0,0 +1,264 @@
+import 'package:fluffychat/pages/chat/chat_input_row.dart';
+import 'package:fluffychat/pages/chat/chat_input_row_style.dart';
+import 'package:fluffychat/pages/chat/chat_view_style.dart';
+import 'package:fluffychat/pages/chat/direct_chat_empty_view.dart';
+import 'package:fluffychat/pages/chat/empty_chat.dart';
+import 'package:fluffychat/pages/chat/input_bar.dart';
+import 'package:fluffychat/resource/image_paths.dart';
+import 'package:fluffychat/utils/adaptive_bottom_sheet.dart';
+import 'package:fluffychat/utils/platform_infos.dart';
+import 'package:fluffychat/widgets/avatar/avatar.dart';
+import 'package:fluffychat/widgets/matrix.dart';
+import 'package:fluffychat/widgets/twake_components/twake_icon_button.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:flutter_gen/gen_l10n/l10n.dart';
+import 'package:keyboard_shortcuts/keyboard_shortcuts.dart';
+import 'package:matrix/matrix.dart';
+import 'package:vrouter/vrouter.dart';
+import 'package:fluffychat/utils/string_extension.dart';
+
+
+class EmptyChatView extends StatelessWidget {
+  const EmptyChatView(this.controller ,{super.key});
+
+  final EmptyChatController controller;
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        toolbarHeight: 64,
+        surfaceTintColor: Colors.transparent,
+        leadingWidth: 8 + 24 + 8,
+        leading: TwakeIconButton(
+          tooltip: "Back",
+          icon: Icons.arrow_back,
+          onPressed: () => VRouter.of(context).pop(),
+          paddingAll: 8.0,
+          margin: const EdgeInsets.symmetric(vertical: 12.0),
+        ),
+        titleSpacing: 0,
+        title: _EmptyChatTitle(
+          receiverId: controller.presentationContact.matrixId!,
+          displayName: controller.presentationContact.displayName,
+        ),
+        actions: _appBarActions(context),
+        bottom: PreferredSize(
+          preferredSize: const Size(double.infinity, 4),
+          child: Container(
+            color: Theme.of(context).colorScheme.surfaceTint.withOpacity(0.08), 
+            height: 1,)),
+      ),
+      body: Column(
+        children: [
+          const Expanded(child: Center(child: DirectChatEmptyView())),
+          Padding(
+            padding: const EdgeInsets.only(bottom: 8.0),
+            child: Row(
+              crossAxisAlignment: CrossAxisAlignment.center,
+              mainAxisAlignment: MainAxisAlignment.spaceBetween,
+              children: [
+                TwakeIconButton(
+                  tooltip: L10n.of(context)!.more,
+                  margin: const EdgeInsets.only(right: 4.0),
+                  icon: Icons.add_circle_outline,
+                  onPressed: () async {
+                    final currentPermissionPhotos = await controller.getCurrentPhotoPermission();
+                    final currentPermissionCamera = await controller.getCurrentCameraPermission();
```

Comment:
Should we do it concurrently?

Reply:
I have tried it, but onlt 1 popup show up, and second time i have to add permission again ;)). i use Future.wait()